app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -342,4 +342,30 @@ class PatientEntryUpdateTest {
         )
   }
 
+  @Test
+  fun `when the age exceeds min limit, then show error`() {
+    //given
+    val errors: List<PatientEntryValidationError> = listOf(PatientEntryValidationError.AGE_EXCEEDS_MIN_LIMIT)
+    val givenModel = defaultModel
+        .fullNameChanged("Name")
+        .ageChanged("0")
```

Comment:
Use the static constant for minimum allowed patient age here?

```kotlin
.ageChanged(MIN_ALLOWED_PATIENT_AGE.minus(1).toString())
```

Reply:
resolved

Reply:
Resolved